docs/index.rst

```diff
@@ -1,4 +1,4 @@
-Welcome to Simularium Conversion's documentation!
+Welcome to simulariumio's documentation!
```

Comment:
I removed a bunch of these docs/*rst files because they are auto-generated and we don't need to track them in git.

Reply:
Thank you! Those have bothered me